ansible/validation.yml

```diff
@@ -0,0 +1,99 @@
+---
+# A series of simple tests to validate the Slurm cluster deployment
+# These will eventually be split into different files to test the 
+# different stages of the deployment.
+# Login and compute groups of slurm cluster have been used as hosts for now.
+
+- hosts: openstack
+  roles:
+    - role: jasmin.cluster-infra
+      cluster_network: "caastest-U-internal"
+      cluster_deploy_user: 'centos'
```

Comment:
I don't think we need this parameter setting, it ought to come from the config yaml

Reply:
So move this to `minimal.yml`?

Reply:
Have you tried removing it here, altogether?  I don't think we define it in other invocations of `jasmin.cluster-infra`

Reply:
It defaults to root, which MPI didn't like.

Reply:
You are quite right.